src/views/Completed.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <ion-page>
+    <ion-header>
+      <ion-toolbar>
+        <ion-buttons slot="start">
+          <ion-menu-button></ion-menu-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content :fullscreen="true">
+      wORKING on Completed Page
```

Comment:
Improve letter case

Reply:
ma'am this is temporary for testing only